### ISSUE TITLE
[UWP] MvvmCross.Plugins.Messenger -> UWP Platform need to be added to portable library

### ISSUE THREAD

**Issue by farlop** [MvvmCross/MvvmCross-Plugins#145]
In order to use this plugin (and maybe others) in UWP, the platform should be enabled at project properties.

**Comment by Cheesebaron** [comment]
What do you mean with "platform should be enabled at project properties"?

**Comment by farlop** [comment]
Yesterday I found troubles trying to use the Messenger plugin in the UWP project. After cloning the repository, configuring 'Windows Universal 10.0' in the core project properties and linking to the compiled ones it started to work again. So I though that it should be something similar to #143 . I'll give it a try again tonight and let you know if it still fails using the current published nuget packages.

**Comment by Cheesebaron** [comment]
Nah, it is probably the PCL profile that is wrong in the nuspec and maybe also wrong profile in the PCL. Will have a look.
